ctfpad/templates/users/edit.html

```diff
@@ -113,17 +105,139 @@ <h5 class="card-title">
+                                            {{ choice }}
+                                        {% endfor %}
+                                    </select>
+                                </div>
+                            {% else %}
+                                <input type="text" name="{{ form.status.html_name }}" value="{{ form.status.value }}"
+                                       hidden/>
+                            {% endif %}
+
+                            <div class="form-row">
+                                <div class="form-group col-md-6">
+                                    <label class="label"><strong>Country</strong></label>
+                                    <div class="input-group mb-3">
+                                        <span class="input-group-text">
+                                            <i class="fas fa-globe-asia"></i>
+                                        </span>
+                                        <select class="form-control" id="{{ form.country.id_for_label }}"
+                                                name="{{ form.country.html_name }}">
+                                            {% for choice in form.country %}
+                                                {{ choice }}
+                                            {% endfor %}
+                                        </select>
+                                    </div>
+                                </div>
+
+                                <div class="form-group col-md-6">
+                                    <label class="label"><strong>Timezone</strong></label>
+                                    <div class="input-group mb-3">
                                         <span class="input-group-text">
                                             <i class="fas fa-clock"></i>
                                         </span>
+                                        <select class="form-control" id="{{ form.timezone.id_for_label }}"
+                                                name="{{ form.timezone.html_name }}">
+                                            {% for choice in form.timezone %}
+                                                {{ choice }}
+                                            {% endfor %}
+                                        </select>
                                     </div>
-                                    <select class="form-control" id="{{ form.timezone.id_for_label }}"
-                                        name="{{ form.timezone.html_name }}">
-                                        {% for choice in form.timezone %}
-                                        {{ choice }}
-                                        {% endfor %}
-                                    </select>
+                                </div>
+                            </div>
+
+                            <label class="label"><strong><abbr title="You can use them to login on the pad directly">Pad
```

Comment:
Is this also a duplicate? Those divs appear twice

Reply:
Very possible. did a lot "Search and Replace" and could be that I missed that!

Reply:
Ahhh I know what happened... That was the merge